tests/integration/docker-compose.yml

```diff
@@ -73,6 +74,31 @@ services:
       AWS_ACCESS_KEY_ID: "user"
       AWS_SECRET_ACCESS_KEY: "password"
 
+  performance-tests:
+    image: grafana/k6
+    command: run -u 20 -i 100 tests/performance/k6_perf.js
+    volumes:
+      - ../../:/src
+      - go-cache:/cache
+      - tmp:/tmp
+    working_dir: /src
+    depends_on:
+      minio:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+      service:
+        condition: service_healthy
+    environment:
+      GOCACHE: /cache/go-build
+      GOMODCACHE: /cache/go-mod
+      FML_SERVICE_URI: http://service:5000
+      FML_DATABASE_URI: ${FML_DATABASE_URI:-postgres://postgres:postgres@postgres/postgres}
```

Comment:
Presumably for this test don't really need these environment vars -- except for FML_SERVICE_URI if used in the js file

Reply:
Thanks, removing them now.